### PR TITLE
[DEX] Open URLs in external browser window

### PR DESCRIPTION
**Pull request by matheusd**
This fixes an issue where a white screen was displayed when attempting to open external links.

### CODE REVIEW

**Comment by bgptr** [comment]
`https://tbtc.bitaps.com/*` links still display a white screen for me.

File: app/main.development.js [hunk 1]
```diff
@@ -428,6 +428,10 @@ function createDexWindow(serverAddress) {
   dexWindow.once("ready-to-show", () => {
     dexWindow.show();
   });
+  dexWindow.webContents.on("will-navigate", (e, url) => {
+    mainWindow.webContents.send("open-external", url);
+    e.preventDefault();
+  });
```
Comment (bgptr) [inline app/main.development.js:434]: So, I noticed that not especially the `https://tbtc.bitaps.com/*` links displayed with a white screen, but links targeted with `_blank`. The `will-navigate `works great with `_self` links, but for `_blank` links I suggest this event handler additionally:
```
  dexWindow.webContents.setWindowOpenHandler(({ url }) => {
    mainWindow.webContents.send("open-external", url);
    return { action: "deny" };
  });
```

Comment (chappjc) [reply app/main.development.js:434]: Just tested and this suggestion works, @bgptr.

The other link was changed to `_blank` in 0.5 BTW, but might as well  handle both targets.

Would be create to get this PR in for the 0.5 DEX update (https://github.com/decred/decrediton/pull/3791).

Comment (chappjc) [reply app/main.development.js:434]: Having tested more, I think the safest thing is to open an external browser for **only** `_blank` target links.  All of the explorer links are now `_blank` targets.  I'm concerned about some normal app navigation clicks going to an external window instead.  I've only seen that happen with "Sign Out" so far though.

Comment (chappjc) [reply app/main.development.js:434]: Yeah, in https://github.com/decred/decrediton/pull/3791 I'm using just `setWindowOpenHandler` and **not** `on("will-navigate", ...)` and it's giving the desired effect.  I've put these commits on that PR branch for testing.

Can we either get this PR revised and merged, or closed and just use the commits included in https://github.com/decred/decrediton/pull/3791?

**Comment by alexlyp** [comment]
Closing in favor of proper changes being included in #3791

**Pull request closed without merging**